moj-docker-deploy/files/ebs-tag.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+
+import boto
+import sys
+import boto.ec2
+import requests
+
```

Comment:
I've amended the salt state (https://github.com/ministryofjustice/moj-docker-deploy-formula/blob/tag-ebs-volumes/moj-docker-deploy/tag-ebs-volumes.sls) to add these pips. Ready for re-review